app/controllers/users/registrations_controller.rb

```diff
@@ -0,0 +1,9 @@
+class Users::RegistrationsController < Devise::RegistrationsController
```

Comment:
__[RuboCop]__

Style/ClassAndModuleChildren: Use nested module/class definitions instead of compact style.


<!-- Comment by [SideCI](https://sideci.com) -->